### DISCUSSION TITLE
Can I include groups to install in path dependencies?

### DISCUSSION THREAD

**Discussion by Newbytee** [python-poetry/poetry#8717] [Q&A]
I have a Poetry project where I have a [path dependency](https://python-poetry.org/docs/dependency-specification/#path-dependencies), which also is managed with Poetry. This path dependency has [optional groups](https://python-poetry.org/docs/managing-dependencies#optional-groups) which contain optional dependencies, and I would like to install one of these optional groups in the top-level project. However, I can't find any way to achieve apart from just copying the list of dependencies to the top-level project, which I would rather not do. Is there any way to achieve what I'd like to do?

**Comment by fgrbr** [comment]
Please have a look at https://github.com/adriangb/python-monorepo/tree/main/poetry for a workaround, where multiple `pyproject.toml` files are being used, e.g., one `pyproject.toml` file for each optional group. Nevertheless, I would also be quite interested in a native solution for this problem.
